janitor/functions.py

```diff
@@ -64,27 +64,27 @@
 )
 
 
-def unionize_dataframe_categories(
-    *dataframes, column_names: Optional[Iterable[pd.CategoricalDtype]] = None
+def unionize_DataFrame_categories(
+    *DataFrames, column_names: Optional[Iterable[pd.CategoricalDtype]] = None
```

Comment:
Same comment as above. `DataFrame` should remain as `dataframe` in functions, methods, and variables.